### DISCUSSION TITLE
CSS and fonts are not working on NextJS production

### DISCUSSION THREAD

**Discussion by indahud** [tailwindlabs/tailwindcss#3627] [Help]
I am using TailwindCSS in my NextJS project and I am using vercel for deployment. I followed TailwindCSS official website guide for installation. However there is one weird issue I am going through. 

CSS and fonts are not working on production if I use seprate component. But they work locally.
If I return NameList `divs` directly in Home function, everything is working perfectly on both locally and on production. Any ideas what is wrong here??
```Javascript
// index.js File
import NameList  from "../designs/NameList"

export default function Home() {
  return (
    <>
      <NameList />
    </>
  )
}


// NameList.js File
const NameList = () => {
  return (
    <div>
      <div className="border-gray-400 border-2">
        <h5 className="max-w-md mb-2 text-3xl font-heading font-extrabold leading-none sm:text-4xl">
          <div className="flex">Name</div>
        </h5>
      </div>
    </div>
  )
}

export default NameList

```

**Comment by Ophelia2** [comment]
This is what I usually do: [UPSers](https://www.upsers.mobi/)

Put the fonts in public/static somewhere

In the same folder as the fonts put a CSS file where the fonts are declared

CSS file example /public/fonts/style.css:

@font-face {
    font-family: 'Italian No1';
    src: url('ItalianNo1-Black.eot');
    src: url('ItalianNo1-Black.eot?#iefix') format('embedded-opentype'), url('ItalianNo1-Black.woff2') format('woff2'), url('ItalianNo1-Black.woff') format('woff');
    font-weight: 900;
    font-style: normal;
}
Import this css in _document.js (docs):

render() {
    return (
        <Html>
            <Head>
                <link href="/fonts/style.css" rel="stylesheet"/>
            </Head>

            <body>
                <Main/>
                <NextScript/>
            </body>
        </Html>
    )
}

**Comment by simonswiss** [comment]
Hi @indahud 👋

I'd need to see more of the code to help, but my bet is that you have the `purge` template paths set incorrectly in your Tailwind config.

Are you able to share your setup (or a minimal reproduction) in a GitHub repo?
